core/templates/pages/profile-page/profile-page.controller.ts

```diff
@@ -55,8 +55,10 @@ angular.module('oppia').directive('profilePage', [
           };
           ctrl.$onInit = function() {
             LoaderService.showLoadingScreen('Loading');
-            $http.get(profileDataUrl).then(function(response) {
-              var data = response.data;
+
+            let fetchProfile = () =>
+              ProfilePageBackendApiService.fetchProfileData();
+            fetchProfile().then(function(data) {
```

Comment:
Had to break this up into 2 lines, if not, the over 80 lines problem would cascade down and cause more linter issues further down